### PR TITLE
Added Effect 'Blimp Strats'

### PR DESCRIPTION
**Pull request by Last0xygen**
Executing the Blimp Strats (Spawning blimp at the "perfect" position and exiting it)
Fix https://github.com/gta-chaos-mod/ChaosModV/issues/758

### CODE REVIEW

**Comment by pongo1231** [comment]
Might want to pause all game scripts until the blimp has crashed into the observatory (`Hooks::EnableFakeTpHook` and `Hooks::DisableFakeTpHook`)

**Comment by Last0xygen** [comment]
Done 👍

File: ChaosMod/Effects/db/Player/PlayerBlimpStrats.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+#include <stdafx.h>
+#include "Memory/Hooks/ScriptThreadRunHook.h"
+
+static void OnStart()
+{
+	Hash blimpHash = GET_HASH_KEY("blimp");
+	LoadModel(blimpHash);
+	Hooks::EnableScriptThreadBlock();
+	Vehicle veh = CreatePoolVehicle(blimpHash, -377.276, 1055.06, 340.962, 80);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerBlimpStrats.cpp:13]: Shouldn't this be a normal vehicle which should be marked as despawnable afterwards instead?

Comment (Last0xygen) [reply ChaosMod/Effects/db/Player/PlayerBlimpStrats.cpp]: Yeah, would be a better idea :D

**Pull request merged**